Tests to find minc toolkit installation don't work as expected

```
root@mincvm:/home/minc/minc2-simple/build# cmake ..
-- The C compiler identification is GNU 7.3.0
-- The CXX compiler identification is GNU 7.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:10 (FIND_PACKAGE):
  Could not find a package configuration file provided by "LIBMINC" with any
  of the following names:

    LIBMINCConfig.cmake
    libminc-config.cmake

  Add the installation prefix of "LIBMINC" to CMAKE_PREFIX_PATH or set
  "LIBMINC_DIR" to a directory containing one of the above files.  If
  "LIBMINC" provides a separate development package or SDK, be sure it has
  been installed.


-- Configuring incomplete, errors occurred!
See also "/home/minc/minc2-simple/build/CMakeFiles/CMakeOutput.log".
root@mincvm:/home/minc/minc2-simple/build# mincstats

Usage: mincstats [options] <infile.mnc>
       mincstats -help


```

Looks like this test doesn't work as expected:
https://github.com/vfonov/minc2-simple/blob/master/CMakeLists.txt#L9

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
 PROJECT(minc2-simple C CXX)
 INCLUDE(CTest)
 
-FIND_PATH( MINC_TOOLKIT_DIR bin/mincstats  HINTS /opt/minc DOC "minc-toolkit install prefix")
+FIND_PATH( MINC_TOOLKIT_DIR bin/mincstats  HINTS $ENV{MINC_TOOLKIT} DOC "minc-toolkit install prefix")
 FIND_PACKAGE( LIBMINC REQUIRED HINTS ${MINC_TOOLKIT_DIR}/lib${LIB_SUFFIX})
 
 OPTION(BUILD_EXAMPLES "Build examples." OFF)

